eclass/coreos-kernel.eclass

```diff
@@ -213,14 +215,38 @@ coreos-kernel_src_install() {
 
 	# build and source must cleaned up to avoid referencing $ROOT
 	rm "${D}/usr/lib/modules/${version}"/{build,source} || die
-	dosym "../../../src/linux-${version}" "/usr/lib/modules/${version}/source"
 
-	# this is just here for linux-info.eclass, mask from prod images
-	dodir "/usr/lib/modules/${version}/build"
-	dosym "../../../../boot/config-${version}" \
-		"/usr/lib/modules/${version}/build/.config"
-
-	save_config build/defconfig
+	# Install a stripped source for out-of-tree module builds (Debian-derived)
+	( \
```

Comment:
This would probably be more readable in a function. I can't think of a need for the subshell otherwise.